src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestWithAcquireLease.java

```diff
@@ -78,7 +78,7 @@ public void should_process_zip_file_only_once_by_processor_thread_which_acquires
         // and
         // We expect only two events one for doc upload and another for doc processed
         List<ProcessEvent> processEvents = processEventRepository.findAll();
-        assertThat(processEvents).hasSize(2);
+        assertThat(processEvents).hasSize(3);
```

Comment:
Comment in line 80 needs updating.
Or consider updating this test to check the items, not just the count.